Automattic/pocket-casts-android
# Transcripts: Display speaker info

## Description
This displays speaker info for transcripts.

## Testing Instructions
1. Load an SRT format transcript with speaker info (e.g. from `Cautionary Tales`)
2. ✅ Notice that speaker info is displayed properly
3. Prioritize order for VTT format in `TranscriptsManagerImpl.supportedFormats` at line 22
4. Install the app and load a VTT format transcript with speaker info (e.g. from `Build your Saas`)
5. ✅ Notice that speaker info is displayed properly
6. Search for speaker name
7. ✅ Notice that speaker names are included in search results and highlighted correctly

## Screenshots or Screencast 

SRT | VTT 
---|---
<img width=300 src="https://github.com/user-attachments/assets/c4fb2c60-7f13-40f9-9eb0-8f07acf42f15"/> | <img width=300 src="https://github.com/user-attachments/assets/3d3eb818-84f6-46fd-a880-855c22a5669e"/>

## Checklist
 
#### I have tested any UI changes...
